src/abaqus.jl

```diff
@@ -564,8 +564,8 @@ function process_output_request(model::Model, solver::Solver, output_request::Ab
             haskey(code_mapping, code) || continue
             field_name = code_mapping[code]
             abbr = get(abbr_mapping, code, code)
-            table = solver(DataFrame, field_name, abbr, solver.time)
-            push!(tables, table)
+            #table = solver(DataFrame, field_name, abbr, solver.time)
+            #push!(tables, table)
         end
```

Comment:
It tries to output nice looking table using DataTables when there's output request in ABAQUS .inp file. Something we maybe don't want to mess into JuliaFEM core code is this whole Abaqus module, we should do own package from this because the idea itself is good but not in our focus at the moment?